### ISSUE TITLE
GAN

### ISSUE THREAD

**Issue by PietroVitiello** [Knee-Deep-in-MRI/DeepSegmentation#20]
Where we are at building the baseline GAN
- [x] Generator
- [x] Discriminator
- [x] Loss functions
- [x] Optimizer
- [x] Training script

**Comment by PietroVitiello** [comment]
Build a functioning GAN for data augmentation of Knee MRI
- [x] Write baseline GAN script
- [x] Test that the model does what it is supposed to
- [ ] Train model with mnist
- [ ] Train model with cifar
- [ ] Train model with actual MRI data

**Comment by PietroVitiello** [comment]
Define metrics for loss
- [x] discriminator loss
- [x] generative loss
- [x] discriminator real and fake accuracy
- [x] print metrics

**Comment by arahosu** [comment]
closing the issue now
